chord_metadata_service/chord/utils.py

```diff
@@ -0,0 +1,26 @@
+from asgiref.sync import async_to_sync
+from rest_framework.request import Request as DrfRequest
+
+from chord_metadata_service.logger import logger
+from chord_metadata_service.discovery.scope import ValidatedDiscoveryScope
+from chord_metadata_service.discovery.types import EntityCountOrBoolResponse
+from chord_metadata_service.discovery.api_views import get_censored_entity_counts_async
+from chord_metadata_service.discovery.utils import get_discovery_data_type_permissions
+
+
+@async_to_sync
+async def get_censored_counts_for_serializer(
```

Comment:
take logger as argument so we can inject a logger with other bindings for nicer structured logging